ga4gh/_protocol_definitions.py

```diff
@@ -1586,3 +1586,29 @@ def __init__(self):
         self.number = None
         self.type = None
         self.value = None
+
+postMethods = \
+    [('/reads/search',
+      'GASearchReadsRequest',
```

Comment:
I think it would be more useful to have the actual classes here, rather than the names. You should just be able to remove the quotes - because this is at the end of the file, all the classes will be defined already, and there won't be any problems.